### PR TITLE
Devel spartan782

### PR DESCRIPTION
**Pull request by spartan782**
Overhauled pipelines and masteriso.sh to build both testing branch and master branches. 
Fixed concourse pipelines and updated output to new home.

### CODE REVIEW

File: concourse/download_minimal_iso.sh [hunk 1]
```diff
@@ -17,4 +17,4 @@
 mkdir -p centos-minimal-iso
 
 # download ISO
-curl -L http://mirrors.usinternet.com/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1804.iso -o centos-minimal-iso/centos-minimal.iso
+curl -L `curl -s https://centos.org/download/ | grep Minimal | awk -F'"' '{print $2}'` -o centos-minimal-iso/centos-minimal.iso
```
Comment (dcode) [inline concourse/download_minimal_iso.sh]: how hard would it be to download this locally?

Comment (spartan782) [reply concourse/download_minimal_iso.sh]: We can do that, question is how do you want to make sure the ISO is up to date on the local download?

Comment (dcode) [reply concourse/download_minimal_iso.sh]: Nightly cron job using `lftp mirror`? Just an idea.

```
lftp -c mirror --delete-first --only-newer --exclude /.*\.torrent/ http://mirror.dal10.us.leaseweb.net/centos/7/isos/x86_64/ ./
```

Comment (spartan782) [reply concourse/download_minimal_iso.sh]: Implementing this on the concourse server. 
30 4 * * * lftp -c mirror --delete-first --only-newer  --exclude=.* --include=CentOS-7-x86_64-Minimal-.*.iso http://mirror.dal10.us.leaseweb.net/centos/7/isos/x86_64/ /srv/centos-minimal

Comment (spartan782) [reply concourse/download_minimal_iso.sh]: scratch that going to implement it on the private mirror to use the dns.

**Comment by spartan782** [comment]
Changes implemented

**Pull request merged**